README.md

```diff
@@ -252,15 +252,17 @@ Check the  [Notification Configuration](#38-notification-configuration) to see h
 
   The EaseProbe would listen on the `0.0.0.0:8181` port by default. And you can access the Live SLA report by the following URL:
 
-  - HTML: `http://localhost:8181/`
+  - HTML: `http://localhost:8181/` ( the query option `?refresh=30s` refreshes the page every 30 seconds))
```

Comment:
? why did we move the refresh back here?

Reply:
this is only for the HTML page, not for the JSON API. so...

Reply:
You can make any change as you like @proditis , I just make the change quickly without thinking better idea...

Reply:
how about these?